app/services/foods_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodsService
+  def get_url(url)
```

Comment:
Most conventional to make these all class methods as it is a utility class rather than a class that needs to hold state (like a Food PORO for example)